src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -197,41 +195,38 @@ public String description() {
      * Returns a post-processing {@code Filter} with the given description and
      * operation.
      *
-     * <p>The {@code description} string describes the returned filter. The
-     * {@link Consumer operation} is the effective implementation of the filter.
-     * It is executed for each {@code HttpExchange} after invoking either the
-     * next filter in the chain or the exchange handler (if this filter is the
-     * final filter in the chain).
+     * <p>The {@link #description() description} describes the returned filter.
+     * The {@link Consumer operation} is the effective implementation of the
+     * filter. It is executed for each {@code HttpExchange} after invoking
+     * either the next filter in the chain or the exchange handler (if this
+     * filter is the final filter in the chain).
      *
      * @apiNote
      * An afterHandler filter is typically used to examine the exchange state
-     * rather than modifying it. The filter operation is executed after
+     * rather than modifying it. The filter {@code operation} is executed after
      * {@link Filter.Chain#doFilter(HttpExchange)} is invoked, this means any
      * subsequent filters in the chain and the exchange handler have been
-     * executed and the response has commonly been sent. The filter is only
-     * expected to send the response in the uncommon case that the exchange
-     * has not been handled before the filter is executed.
+     * executed. The filter {@code operation} is not expected to handle the
+     * exchange or {@linkplain HttpExchange#sendResponseHeaders(int, long) send the response headers}.
+     * Doing so is likely to fail, since this is commonly done by the exchange
+     * handler.
```

Comment:
maybe `... since the request is expected to have already been  handled before the operation is executed`?